polyglot/piranha/src/models/piranha_output.rs

```diff
@@ -36,4 +38,12 @@ impl PiranhaOutputSummary {
   pub(crate) fn rewrites(&self) -> &[Edit] {
     self.rewrites.as_ref()
   }
+
+    pub fn path(&self) -> PathBuf {
```

Comment:
Indentation? This seems more deeply indented than the function before it. Are you mixing tabs and spaces?

Reply:
Yes I think so. I will run `cargo format` and `clippy` next.